app.js

```diff
@@ -31,7 +31,8 @@ router.get('/people', function(req, res) {
         {name: 'joonhan', url: 'http://cse.postech.ac.kr/eng/wp-content/uploads/2017/05/professor18-168x191.jpg'},
         {name: 'mscha', url: 'assets/mscha.jpg'},
         {name: 'leesy', url: 'http://cse.postech.ac.kr/eng/wp-content/uploads/2017/05/professor12-168x191.jpg'},
-        {name: 'woo', url: 'assets/woo.jpeg'}        
+        {name: 'woo', url: 'assets/woo.jpeg'},
+        {name: 'ko', url: 'assets/postech.jpg'}  
```

Comment:
Can we add a comma at the end, so the next person doesn't have to edit this line?